__tests__/api/login/route.js

```diff
@@ -58,4 +73,14 @@ describe("POST", () => {
             }
         });
     });
+    it("should successfully call Redis", async ()=> {
+        const requestMock = {
+            json: async () => (testRequestPayload),
+        };
+        const LoginResponse = await POST(requestMock);
```

Comment:
nit: this isn't really a mockRequest so we could probably change it's name accordingly
```suggestion
        const request = {
            json: async () => (testRequestPayload),
        };
        const LoginResponse = await POST(request);
```